apps/balancer-tools/src/components/Header.tsx

```diff
@@ -27,12 +28,14 @@ export function Header({
   title,
   children,
   endButton = <CustomConnectButton />,
+  onLinkClick,
 }: IHeader) {
   return (
     <div className="flex h-20 w-full items-center border-b border-b-blue3 bg-blue2 p-4 text-white">
       <div className="mr-auto flex flex-1 justify-start">
         <Link
           href={linkUrl}
+          onClick={onLinkClick}
           className="flex items-center gap-3 justify-self-start"
         >
           {imageSrc && <Image src={imageSrc} height={50} width={50} alt="" />}
```

Comment:
Think we should probably make this composable. wdyt @luizakp ? We'd have something like `HeaderLink` and use it in children or pass it as a prop. IMO the best approach is to take both this link and the EndButton away and just have children to have a clearer API 😅